### DISCUSSION TITLE
NFC reader module?

### DISCUSSION THREAD

**Discussion by gbdlin** [FrameworkComputer/inputmodule-rs#41] [Ideas]
I wonder if it's possible to create a NFC module that would be mounted instead of a blank module on either side of the touchpad? Obviously it would work next to the keyboard, but not sure if there are any contacts exposed for it to occupy the space around the touchpad?

**Comment by JohnAZoidberg** [comment]
Possible yeah, next to the keyboard. Same shape as numpad or LED matrix.
You can find more info about the hardware here: https://github.com/FrameworkComputer/InputModules/

Next to the touchpad, no. The touchpad row on the bottom is I2C to the embedded controller. And all 3 locations where you can put the touchpad are wired together. So you cannot use anything else next to the touchpad.
